console/lib/media/api.dart

```diff
@@ -149,6 +149,24 @@ abstract class media {
         });
   }
 
+  static Future<GenerateTorrentResponse> publishable(
```

Comment:
rename GenerateTorrent -> Published in the protbuf.
```suggestion
  static Future<PublishedResponse> publishable(
```